exercises/practice/gigasecond/test/gigasecond_test.dart

```diff
@@ -2,49 +2,19 @@ import 'package:gigasecond/gigasecond.dart';
 import 'package:test/test.dart';
 
 void main() {
-  group("Gigasecond", gigasecondTests);
-}
-
-void gigasecondTests() {
-  group('Add one gigasecond to the input. ', () {
-    test('date only specification of time', () {
-      final DateTime birthDate = DateTime.utc(2011, DateTime.april, 25);
-      final DateTime result = add(birthDate);
```

Comment:
The name of the `add` function is very generic. It might break tests for the existing solutions, but I think a more appropriate name would be `addGigaSecond` or `addGigaSecondTo`.

Reply:
I like `addGigaSecondTo`. Very explicit and nice! I'll go ahead and change that.